scripts/pylib/twister/twisterlib/reports.py

```diff
@@ -354,6 +354,9 @@ def json_report(self, filename, version="NA", platform=None, filters=None):
             elif instance.status == TwisterStatus.SKIP:
                 suite["status"] = TwisterStatus.SKIP
                 suite["reason"] = instance.reason
+            else:
```

Comment:
`elif instance.status == TwisterStatus.NOTRUN:`, not just `else`. `else` should be used to indicate an unknown status.

Reply:
done